demos/sdk_output_skeleton_13_buildings/run/baseline_scenario/ghe_dir/sys_params_proportional.json

```diff
@@ -597,6 +597,10 @@
       },
       "central_pump_parameters": {
         "pump_design_head": 60000
+      },
+      "wahp": {
+        "cop_c": 3.0,
+        "cop_h": 3.0
```

Comment:
https://github.com/urbanopt/geojson-modelica-translator/pull/640 implements automatic inclusion of these fields when using the `uo_des` cli (which gets called from the SDK cli).